spin-observatory-plugin-deck/src/components/pipelines/ExecutionsTable.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+  Paper,
+  Table,
+  TableBody,
+  TableContainer,
+  TableFooter,
+  TableRow,
+  TablePagination,
+  Typography,
+} from '@material-ui/core';
+import React, { ChangeEvent, useState } from 'react';
+import { IExecution } from '@spinnaker/core';
+import { TableHeaders } from './TableHeaders';
+import { ExecutionRow } from './ExecutionRow';
+import { IStatus, STATUSES, DEFAULT_ROWS_PER_PAGE } from './constants';
+import { PaginationActions } from './PaginationActions';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
```

Comment:
You don't like stylesheets? lol

Reply:
just felt like there were few enough style changes that it made sense to keep them in the same files